### ISSUE TITLE
Headsup: The new SDK (js-browserless) supports React-Native

### ISSUE THREAD

**Issue by felixspitzer** [lemonde/react-native-at-internet#8]
We have been using a fork of your repo to send our events to ATInternet (now Piano Analytics) and are grateful for the help the repo has provided.

In return we would like to mention, that the new Piano Analytics JS-browserless SDK supports React Native out of the box. So the native SDKs are not the only solution for RN anymore.

Maybe the package could mention this option for developers who are looking to implement Piano Analytics into their app.

The main purpose of this request is to thank you for the repo, that we have used! ;)

**Comment by riamon-v** [comment]
Hello @felixspitzer, would you be able to provide an example of an app working with the npm package `piano-analytics-js` ?
I personnaly get the following errors.

<img width="860" alt="Screenshot 2022-10-31 at 11 28 25" src="https://user-images.githubusercontent.com/19188004/198992614-69d36d26-bbc9-4e88-a00e-1b1993f8970e.png">

I tried putting the `setConfigurations` function into the `index.js`of my app. Wherever I import the module I get this

**Comment by felixspitzer** [comment]
@riamon-v 
I will provide one by the end of the week :)

**Comment by riamon-v** [comment]
> @riamon-v I will provide one by the end of the week :)

@felixspitzer  Yes it would be awesome. I installed version 6.5.0 of `piano-analytics-js`as it was the first to support RN, but it doesn't seem to change anything.
Looking forward to your reply

**Comment by felixspitzer** [comment]
@riamon-v 

Please have a look at this. Just fill in your site ID and your collect domain and click fire away with the button. It is a very basic setup, but the events reach the Piano backend and can be viewed there.

If you have further questions, feel free to ask :D I am not associated with Piano, but we use the SDK in our projects. So I might be able to help :)

https://github.com/felixspitzer/PianoExampleRN

**Comment by riamon-v** [comment]
Well the problem in my app is actually that the project doesn't seem to detect it is a RN project. So it imports the web part of the package.
To fix this I directly imported piano-analytics like so:
`import {pianoAnalytics} from 'piano-analytics-js/dist/browserless/piano-analytics.react-native.umd';`

Hope it can help some that will encounter the same problem :D
